include/Game.hpp

```diff
@@ -15,6 +15,7 @@
 # include "Renderer.hpp"
 # include "Keyboard.hpp"
 # include "Mouse.hpp"
+# include "Joystick.hpp"
```

Comment:
The contents of this include seem unused in `Game.hpp`.
consider moving it to the .cpp(s) where it is used.